fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -2580,6 +2582,97 @@ module.exports = function(config, DB) {
       })
     })
 
+    describe('db.verifyTokensShortCodes', () => {
```

Comment:
nit: "ShortCodes" plural in the name here doesn't seem quite right, should it be singular?